packages/core/src/system.ts

```diff
@@ -91,6 +106,8 @@ export interface ActorSystem<T extends ActorSystemInfo> {
 
 export type AnyActorSystem = ActorSystem<any>;
 
+const rootSubscriptionKey = '@xstate.system.root' as const;
```

Comment:
added a root subscription key that seemed unique enough while following some conventions I found.

This would be used in the case `actor.system.subscribe(event => ...)` was used with no `systemId`.